web/src/components/Plugin/UI/authz-casbin.tsx

```diff
@@ -37,54 +38,122 @@ const FORM_ITEM_LAYOUT = {
 const AuthzCasbin: React.FC<Props> = ({ form, schema }) => {
   const { formatMessage } = useIntl();
   const properties = schema?.properties
+  const [value, setValue] = useState(1);
+  const onChange = (e:any) => {
+    setValue(e.target.value);
+  };
+
+  const [dian, setDian] = useState('houmian')
+  const aa = () => {
+    setDian('houmian')
+  }
+
+  const bb = () => {
+    setDian('qianmian')
+  }
 
   return (
     <Form form={form} {...FORM_ITEM_LAYOUT}>
-      <Form.Item
-        name="model_path"
-        label="model_path"
-        rules={[
-          {
-            required: true,
-            message: `${formatMessage({ id: 'component.global.pleaseEnter' })} model_path`,
-          },
-        ]}
-        initialValue={properties.model_path.default}
-        tooltip={formatMessage({ id: 'component.plugForm.authz-casbin.model_path.tooltip' })}
-      >
-        <Input />
-      </Form.Item>
-      <Form.Item
-        name="policy_path"
-        label="policy_path"
-        rules={[
-          {
-            required: true,
-            message: `${formatMessage({ id: 'component.global.pleaseEnter' })} policy_path`,
-          },
-        ]}
-        initialValue={properties.policy_path.default}
-        tooltip={formatMessage({ id: 'component.pluginForm.authz-casbin.policy_path.tooltip' })}
-      >
-        <Input />
-      </Form.Item>
+      <Radio.Group onChange={onChange} value={value}>
+        <Radio value={1} onClick={aa}>A</Radio>
+        <Radio value={2} onClick={bb}>B</Radio>
+      </Radio.Group>
+      {dian === 'houmian' ?
```

Comment:
```suggestion
      {dian === 'houmian' ?
```
Bad variable names, and we can use `value` instead of `dian` to make a judgement